docs/cloud/satellites/byoc/aws/terraform.md

```diff
@@ -0,0 +1,64 @@
+# Installing BYOC in AWS Using Terraform
+
+This page documents the requirements and steps required to install [BYOC Satellites](../byoc.md) in AWS using Earthly's Terraform module.
+
+## Requirements
+Before you begin to provision your BYOC configuration, ensure that you meet the [base requirements](./requirements.md) for installation within AWS.
+
+There are also a few additional requirements you will need to make sure you meet:
+
+* An AWS account or role that can create all the resources specified in the module.
+* You have permission to list Terraform outputs.
+
+## Using the BYOC Terraform Module
+You can find the module in the [public Terraform Registry](https://registry.terraform.io/modules/earthly/byoc/aws/latest). Here is an example of using the module:
+
+```hcl
+module "byoc" {
+  source  = "earthly/byoc/aws"
+  version = "0.0.8"
+
+  cloud_name = "my-cloud"
+  subnet = "subnet-0123456789abcde01"
+}
+```
+### Module Inputs
+
+| Name           | Description                                                                                                                                              |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cloud_name     | The name to use to identify the cloud installation. Used by Earthly during automatic installation, and to mark related resources in AWS.                 |
+| subnet         | The subnet Earthly will deploy satellites into.                                                                                                          |
+| ssh_public_key | (Optional) The SSH key to include in provisioned satellites. If left unspecified, a new key is generated, and the private key is available as an output. |
```

Comment:
Are we going to support changing the address resolution from IP to DNS here too?

Reply:
no